content/en/serverless/step_functions/troubleshooting.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Troubleshooting Serverless Monitoring for AWS Step Functions
+kind: documentation
+---
+
+## I cannot see any traces
+
+### Verify that your Step Function is configured to send all logs
+- Ensure that the `DD_TRACE_ENABLED` environment variable is set to `true`.
+- In your AWS console, open your Step Function's logging tab. Ensure that _Log level_ is set to `ALL`, and that _Include execution data_ is selected.
+- Ensure that the CloudWatch log group (also found on the logging tab) has a subscription filter to the Datadog Lambda Forwarder in the same region.
+
+### Verify that logs are forwarded successfully to Datadog
+- Check the Datadog Lambda Forwarder for error messages. Ensure that you have correctly set your API key and Datadog site.
+- Enable `DEBUG` logs on the Datadog Lambda Forwarder by setting the environment variable `DD_LOG_LEVEL` to `debug`.
+
+### Verify that logs are searchable on Live Search
+In Datadog, go to [*Logs*][1] > [*Live Tail*][2]. Search for `source:stepfunction`. You may need to trigger the state machine a few times.
```

Comment:
For consistency, it's bolded in the section below. Also, since the end goal is to send user to Live Tail, I recommend cutting out the link for the Logs page.
```suggestion
In Datadog, go to [**Logs > Live Tail**][2]. Search for `source:stepfunction`. You may need to trigger the state machine a few times.
```